Website/ui/src/modules/Settings/PluginSettings.vue

```diff
@@ -32,6 +34,21 @@ export default {
         plugins: {
             type: Array,
             required: true
+        },
+        mainSettings: {
+            type: Object,
+            required: true
+        }
+    },
+    computed: {
+        enriched_plugins: function() {
```

Comment:
Let's use camelCase every where we can. There could be forgotten samples as snake_case, It would be great if we could refactor them.

Reply:
Done, as far as the code changed in this PR is concerned.

I like the idea of changing it everywhere. But let's do that in a separate change.